phase2_start_service_processes() called -- currently a NOP
phase3_start_service_processes() called -- currently a NOP
phase4_start_service_processes() called -- currently a NOP
phase5_start_service_processes() called -- currently a NOP
testcase_main(): started
QUICK SUMMARY: Attempt to call zap() on a process that has already terminated, but whose parent has not yet called join().
XXp1(): started
XXp1(): arg = `XXp1'
XXp2(): started
XXp2(): arg = `XXp2'
XXp2(): exiting by calling quit(2)
XXp1(): after fork of child 5
XXp3(): started
XXp3(): arg = `XXp3'
XXp3(): calling zap(5)
ERROR: Attempt to zap() a process that is already in the process of dying.
TESTCASE ENDED: Call counts:   check_io() 0   clockHandler() 0
